hpx/runtime/threads/threadmanager_impl.hpp

```diff
@@ -296,6 +296,12 @@ namespace hpx { namespace threads
             pool_.reset_thread_distribution();
         }
 
+	// Returns the underlying scheduling policy
+        scheduling_policy_type& get_pool_sched() const
+	{
+	    return pool_.get_sched();
+	}
+
```

Comment:
Sure, I will fix the naming.  
Yes we can get access to any other scheduler via this API. But I think  it will not require the other schedulers expose the same functionality (i.e. get_disabled_os_threads ), as the scheduler name can be checked via get_scheduler_name() before using get_disabled_os_threads.